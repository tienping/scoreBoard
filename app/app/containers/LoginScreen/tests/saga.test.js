/**
 * Test  sagas
 */

/* eslint-disable redux-saga/yield-effects */
// import { take, call, put, select } from 'redux-saga/effects';
import  defaultSaga  from '../saga';

describe('defaultSaga Saga', () => {
    it('Expect to have unit tests specified', () => {
        let generator = defaultSaga();
        const selectDescriptor = generator.next().value;
        expect(selectDescriptor).toMatchSnapshot();

    });
});
