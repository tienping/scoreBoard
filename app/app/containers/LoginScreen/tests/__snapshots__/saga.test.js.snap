// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultSaga Saga Expect to have unit tests specified 1`] = `undefined`;
